pages/pipelines/hosted_agents/terminal_access.md

```diff
@@ -0,0 +1,44 @@
+# Hosted agents terminal access
+
+The Buildkite hosted agents feature provides you with _terminal/console access_ to jobs running on hosted agents. This feature is useful in allowing you to:
+
+- Understand what components are installed, as you set up your pipeline.
+- Test the behavior of different scripts (because they may not be well-documented).
+- Debug issues that are not reproducible in your local environment.
+
+This can be useful when migrating your pipelines across to [queues](/docs/clusters/manage-queues) on Buildkite hosted agents.
+
+## Use terminal access on hosted agents
+
+Assuming that [terminal access is active across your Buildkite organization](#deactivate-and-reactivate-terminal-access-on-hosted-agents), you can access this terminal access feature from a currently building pipeline, when the job of the relevant step is being built.
+
+The terminal access feature is available to users who have/are any of the following:
+
+- build permissions on the pipeline that created the job
+- a [maintainer of the cluster](/docs/clusters/manage-clusters#manage-maintainers-on-a-cluster) containing this pipeline
+- a Buildkite organization administrator of this cluster
```

Comment:
```suggestion
The terminal access feature is available to users who have:

- build permissions on the pipeline that created the job

and one of the following:

- a [maintainer of the cluster](/docs/clusters/manage-clusters#manage-maintainers-on-a-cluster) containing this pipeline
- a Buildkite organization administrator of this cluster
```

Reply:
Actually, from my own testing, this isn't true...

> build permissions on the pipeline that created the job

I set up a user account _with only this permission_ and I was able to access the terminal.